index.js

```diff
@@ -63,5 +63,6 @@ module.exports.plugins = [
     ]
   ],
   [require("remark-lint-strong-marker"), "*"],
-  [require("remark-lint-table-cell-padding"), "padded"]
+  [require("remark-lint-table-cell-padding"), "padded"],
+  [require("remark-lint-no-consecutive-blank-lines")]
```

Comment:
I think the rules are all in alphabetical order. Can you move this to where it belongs alphabetically?

(Probably makes sense to add a comment about this. Feel free to do that too if you want.)

Reply:
Does't look like they were totally sorted, but it makes sense so I did that in a separate commit